app/editor/src/features/home/InfoText.tsx

```diff
@@ -1,4 +1,20 @@
+import { defaultAlert } from 'features/admin/alerts/constants';
+import React from 'react';
+import { useAlerts } from 'store/hooks/admin';
+import { IAlertModel } from 'tno-core';
+
 export const InfoText: React.FC = () => {
+  const [, api] = useAlerts();
+  const [alert, setAlert] = React.useState<IAlertModel>(defaultAlert);
+
+  React.useEffect(() => {
+    api.findAllAlerts().then((data) => {
```

Comment:
Only fetch one record (the latest enabled message).  I know it will only have one record, but this implementation is not intuitive and will result in maintenance issues.